frontend/src/app/testResults/uploads/CsvUploadGuide.tsx

```diff
@@ -0,0 +1,271 @@
+import { useDocumentTitle } from "../../utils/hooks";
+
+/* eslint-disable jsx-a11y/anchor-has-content */
+const CsvUploadGuide = () => {
+  useDocumentTitle("CSV Upload Guide");
+  return (
+    <div className="prime-home">
+      <div className="grid-container">
+        <div className="prime-container card-container">
+          <div className="tablet:grid-col-8 usa-prose">
+            <section id="anchor-top">
+              <span className="text-base text-italic">
+                Updated: January 2022
+              </span>
+              <h2 className=" margin-top-0">
+                CSV upload guide{" "}
+                <span className="text-secondary bg-white border-1px border-secondary font-body-3xs padding-x-1 padding-y-05 text-secondary margin-left-2 text-ttbottom">
+                  Pilot program{" "}
+                </span>
+              </h2>
+              <p className="usa-intro text-base margin-bottom-4">
+                Step-by-step instructions and guidance for preparing and
+                uploading test results via a comma-separated values (CSV) file.
+              </p>
+              <p>
+                Use a simple online tool to submit a CSV file with a{" "}
+                <a
+                  href="/getting-started/testing-facilities/csv-schema"
+                  className="usa-link"
+                >
+                  standard schema
+                </a>
+                . Receive real-time validation and feedback on file format and
+                field values before submission.
+              </p>
+              <div className="usa-alert usa-alert--info margin-y-6">
+                <div className="usa-alert__body">
+                  <h3 className="usa-alert__heading font-body-md margin-top-05">
+                    About CSV upload
+                  </h3>
+                  This feature is currently being piloted in select
+                  jurisdictions with organizations or facilities that have
+                  existing Electronic Medical Record (EMR) systems. Pilot
+                  partners are selected by recommendation from jurisdictions.
+                  Find out if your jurisdiction is{" "}
+                  <a href="/how-it-works/where-were-live" className="usa-link">
+                    partnered
+                  </a>{" "}
+                  with SimpleReport and{" "}
+                  <a
+                    href={
+                      "mailto:" +
+                      "TODO" +
+                      "?subject=Getting started with SimpleReport"
+                    }
+                    className="usa-link"
+                  >
+                    contact us
+                  </a>{" "}
+                  to learn more.
+                </div>
+              </div>
+              <p>
```

Comment:
Would recommend removing both of these sections, at least for the launch. There's a lot of content before "In this guide," and the initial users probably don't need information about what CSV uploader is or how to get signed up.

Reply:
Done 👏 [afb0b7d](https://github.com/CDCgov/prime-simplereport/pull/3928/commits/afb0b7daff592ea23efc832618b88235dd03cac0)